### PR TITLE
Added channels from the SPR

### PR DESCRIPTION
**Pull request by GerardoHD**

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -13,6 +13,12 @@ http://stream2.dynalias.com:1935/live/tvlive1/playlist.m3u8
 http://stream2.dynalias.com:1935/live/tvlive1/playlist.m3u8?GrupoTopIptv
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/g9zTQcC.jpg" group-title="Local",CANAL 15 Zacatecas
 http://wowza.zuperdns.net:1935/zuperhosting.live.grupob.canal20zacatecas/smil:livestream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/giqLRn8.png" group-title="",Canal 22.1
+https://5e50264bd6766.streamlock.net/canal22/smil:canal22.smil/chunklist_w1978382888_b978000_sleng.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/U2JblvH.png" group-title="",Canal 22.2
+https://5e50264bd6766.streamlock.net/canal222/smil:canal222.smil/chunklist_w786836973_b978000_sleng.m3u8
```
Comment (freearhey) [inline channels/mx.m3u:19]: Links do not work.

File: channels/mx.m3u [hunk 2]
```diff
@@ -69,3 +79,5 @@ http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u
 http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/bjGgLrA.jpg" group-title="",Telemax
 http://s5.mexside.net:1935/telemax/telemax/.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/15kcNRb.png" group-title="", TVMÁS
+https://5ca9af4645e15.streamlock.net/rtv/videortv/chunklist_w2116443559.m3u8
```
Comment (freearhey) [inline channels/mx.m3u:83]: Link do not work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/mx.m3u (hunk 1 above)
Comment (GerardoHD) [reply channels/mx.m3u:19]: Updated URLs:

#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/giqLRn8.png" group-title="",Canal 22.1
https://5e50264bd6766.streamlock.net/canal22/smil:canal22.smil/playlist.m3u8
#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/U2JblvH.png" group-title="",Canal 22.2
https://5e50264bd6766.streamlock.net/canal222/smil:canal222.smil/playlist.m3u8

File: channels/mx.m3u (hunk 2 above)
Comment (GerardoHD) [reply channels/mx.m3u:83]: Updated link:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/15kcNRb.png" group-title="", TVMÁS
https://5ca9af4645e15.streamlock.net/rtv/videortv/chunklist_w192674072.m3u8
```
The other day i saw that those three links stop working by a day or two, and then resume working again.

**Review by freearhey — APPROVED** [review]

**Pull request merged**